src/Syntax.jl

```diff
@@ -174,19 +217,23 @@ Contains the elements that make up the Stock and Flow block syntax.
               `sum_name = [a, b, c, ...]`
 """
 struct StockAndFlowBlock
-    stocks::Vector{Symbol}
-    params::Vector{Symbol}
+    stocks::Vector{StockArgT}
```

Comment:
Is the docstring for this data type still accurate re: stocks & params?

Reply:
Good point, needs updating, will do so.

It general it would be a moral good to update formatting, docstrings, documentation, etc. for the whole project.  Need to replace all the four spaces with two spaces at some point.